### ISSUE TITLE
WISH: cross-compile in devmode

### ISSUE THREAD

**Issue by hiiamboris** [red/REP#86]
Right now `-c -t (other OS)` works like `-r`, and this takes a while.
Would be nice to support fast cross compilation.

**Comment by greggirwin** [comment]
I agree this would be nice. Do we put the onus on the user to make sure they target different dirs, so the runtimes stay separate?

**Comment by hiiamboris** [comment]
I think so. And the most likely combo `.dll` & `.so` won't cause conflict anyway.
